CHANGELOG.md

```diff
@@ -3,6 +3,9 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+### Added
+- Ability to add tooltips to `Slider` and `RangeSlider`, which can be visible always or on hover. Tooltips also take a position argument. [#564]
```

Comment:
This doesn't work quite like in conversations - you need to put the full link. (I notice there's a missing left paren below, would you mind fixing that too?)